packages/sdk/src/vdp/pipeline/PipelineClient.ts

```diff
@@ -113,6 +117,62 @@ export class PipelineClient extends APIResource {
     }
   }
 
+  async listPaginatedNamespacePipelineRuns(
+    props: ListPaginatedNamespacePipelineRunsRequest,
+  ) {
+    const { pipelineName, fullView, pageSize, page, orderBy, filter } = props;
```

Comment:
What do you mean by this?

Reply:
@orangecoloured You are expose a custom parameter fullView, but our API is exposing the view parameter. SDK should be a mirror mirroring exactly what API is doing to stay consistent. Or it will be hard for other user to use our SDK. 

So my suggestion is we need to replace `fullView` flag with `view` parameter. And you can make your viewFull flag happening in the react-query-service